app/views/hyrax/base/_workflow_actions.html.erb

```diff
@@ -1,32 +1,40 @@
 <div id="workflow_controls" class="panel panel-workflow">
   <div class="panel-heading">
     <a data-toggle="collapse" href="#workflow_controls_collapse">
-      <h2 class="panel-title">Review and Approval <i class="fa fa-chevron-right pull-right"></i></h2>
+      <h2 class="panel-title"><%= t('hyrax.base.items.workflow.review') %> <i class="fa fa-chevron-right pull-right"></i></h2>
     </a>
   </div>
   <%= form_tag main_app.hyrax_workflow_action_path(presenter), method: :put do %>
     <div id="workflow_controls_collapse" class="panel-body panel-collapse collapse">
       <div class="col-sm-3 workflow-actions">
-        <h3>Actions</h3>
+        <h3><%= t('hyrax.base.items.workflow.action') %></h3>
 
         <% presenter.workflow.actions.each do |key, label| %>
           <div class="radio">
             <label>
               <%= radio_button_tag 'workflow_action[name]', key, key == 'comment_only' %>
-              <%= label %>
+              <%= t('hyrax.base.items.workflow.actions.' + key) %>
             </label>
           </div>
         <% end %>
       </div>
       <div class="col-sm-9 workflow-comments">
-        <div class="form-group">
-          <label for="workflow_action_comment">Review comment:</label>
-          <textarea class="form-control" name="workflow_action[comment]" id="workflow_action_comment"></textarea>
-        </div>
+          <% if flash['alert'].present? %>
```

Comment:
Might there be any other possible way to get to this page and have an `flash['alert']` that has nothing to do with this?

Reply:
I'm not sure. It doesn't seem that the `errors` are available on that page in the presenter... Open to suggestion. I could check for the inclusion of "comment" in the alert or something.

Reply:
Yeah, I don't have a good suggestions.  Since it's a redirect, there's no errors available.